lib/ui/onboarding/check_phrase_page.dart

```diff
@@ -7,11 +8,13 @@ class CheckPhrasePage extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      bottomNavigationBar: HyphaAppButton(
-        onPressed: () {
-          // Get.to(() => CreateAccountSuccessPage(), transition: Transition.rightToLeft);
-        },
-        title: 'Next',
+      bottomNavigationBar: HyphaSafeBottomNavigationBar(
+        child: HyphaAppButton(
+          onPressed: () {
+            // Get.to(() => CreateAccountSuccessPage(), transition: Transition.rightToLeft);
```

Comment:
it was commented before but why is this commented out?

Reply:
This page is unused.